src/main/java/de/tum/in/www1/artemis/repository/ExamRepository.java

```diff
@@ -513,4 +513,12 @@ private static Map<Long, Integer> convertListOfCountsIntoMap(List<long[]> examId
                 AND registeredUsers.user.id = :userId
             """)
     Set<Exam> findActiveExams(@Param("courseIds") Set<Long> courseIds, @Param("userId") long userId, @Param("visible") ZonedDateTime visible, @Param("end") ZonedDateTime end);
+
+    @Query("""
+            SELECT COUNT(e) > 0
+            FROM Exam e
+            WHERE e.id = :examId
+                AND e.testExam
```

Comment:
imo a bit more readable.
```suggestion
            SELECT e.testExam
            FROM Exam e
            WHERE e.id = :examId
```